apps/web/modules/bookings/views/bookings-listing-view.getStaticProps.tsx

```diff
@@ -1,24 +0,0 @@
-import { type GetStaticProps } from "next";
```

Comment:
To my understanding:
- Cal.com used SSG to pre-build routes only for English locale at build time
- which is no longer needed because we build only English locale route for every route by default now